Network/dedicated-load-balancer-firmware-upgrade.md

```diff
@@ -0,0 +1,231 @@
+{{{
+  "title": "Dedicated Load Balancer Firmware update",
+  "date": "12-11-2015",
+  "author": "Maxim Volkov",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+### Introduction
+
+To address existing bugs and bring to light set of new features periodically a new firmware for your virtual load balancer is released. To make those improvements available you would need to go through the upgrade  procedure. In case of HA pair upgrade no down time would be required, however the proper planning is the key. The high level preview of the upgrade process looks as follows - disconnect Secondary node from synchronization, upgrade it, convert into Primary node; disconnect original Primary node from synchronization, upgrade it, turn on synchronization, convert into Primary node again.
+
+### Prerequisites
+
+* you have HA dedicated load balancers which have been configured under your account
+* you have ability to login into appliance with administrative credentials
+
+### Preparation
+
+First of all, you would need to create a ticket and ask NOC staff to upload the version of the firmware you are interested in upgrading to.
```

Comment:
assume that the reader isn't sure how to open a ticket with the NOC.  provide a link to how to submit a support request.  you may even want to include a code snippet to copy and paste for the email of the body like ecosystem does and we do (see https://www.ctl.io/knowledge-base/network/centurylink-cloud-guide-to-openvpn-access-server/#reserving-client-vpn-dhcp-pool-in-the-control-portal for example)